homeassistant/components/sensoterra/models.py

```diff
@@ -0,0 +1,13 @@
+"""Sensoterra models."""
+
+from enum import StrEnum, auto
+
+
+class ProbeSensorType(StrEnum):
```

Comment:
Move this to `sensor.py`. You only use every type once, so maybe its time to ask ourselves how much this adds